ibis/expr/rules.py

```diff
@@ -149,8 +149,6 @@ def dtype(self):
         dtypes = [arg.dtype for arg in args]
         if util.all_of(dtypes, dt.Integer):
             result = _promote_integral_binop(args, op)
-        elif util.all_of(dtypes, dt.Decimal):
-            result = _promote_decimal_binop(args, op)
```

Comment:
Can you remove the now unused `_promote_decimal_binop`?

Reply:
Done!